src/SolutionAssemblyInfo.cs

```diff
@@ -1,38 +0,0 @@
-﻿// --------------------------------------------------------------------------------------------------------------------
```

Comment:
It only supplied two useful attributes (copyright notices). The versioning info is all provided by the build system via NB.GV (can't GitVersion do this too?). And the copyright attributes only need to appear in one or two assemblies anyway. 
I can put it back though, for the sake of the two attributes, but FWIW I removed it because I like projects to be super-familiar to users and feel normal unless there is a strong benefit from doing otherwise, and to avoid redundancy of two attributes across two projects, it didn't seem worth it in my estimation.